tests/server.py

```diff
@@ -0,0 +1,20 @@
+"""Test OpenDAP server with air temperature dataset."""
+import numpy as np
+import xarray.tutorial
+import xpublish
+
+from xpublish_opendap import OpenDapPlugin
+
+ds = xarray.tutorial.open_dataset("air_temperature")
+
+ds_attrs_quote = xarray.tutorial.open_dataset("air_temperature")
+ds_attrs_quote.attrs["quotes"] = 'This attribute uses "quotes"'
+ds_attrs_quote.attrs["npint"] = np.int16(16)
+ds_attrs_quote.attrs["npintthirtytwo"] = np.int32(32)
+
+rest = xpublish.Rest(
+    {"air": ds, "attrs_quote_types": ds_attrs_quote},
```

Comment:
Can we split casting and escaping attrs into separate datasets and tests?

```suggestion
ds_attrs_cast = xarray.tutorial.open_dataset("air_temperature")
ds_attrs_cast.attrs["npint"] = np.int16(16)
ds_attrs_cast.attrs["npintthirtytwo"] = np.int32(32)

rest = xpublish.Rest(
    {"air": ds, "attrs_quote_types": ds_attrs_quote, "attrs_cast": ds_attrs_cast},
```

Even though they happen in the same functions, they are different enough that they are likely different enough to fail in different ways. 

If we split them into separate datasets and tests, then we can tell which failed in the future when someone like me starts messing with that function tries to clean things up while forgetting why it's ended up the way it is.